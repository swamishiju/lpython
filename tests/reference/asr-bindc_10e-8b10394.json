{
    "basename": "asr-bindc_10e-8b10394",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/bindc_10e.py",
    "infile_hash": "fa023d4ad17e426f778a99326998e6dbeac5428fa252974a39a3219d",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-bindc_10e-8b10394.stderr",
    "stderr_hash": "bcf748808d51f207b43bff022694c043355035c59702c19af6a956ba",
    "returncode": 2
}