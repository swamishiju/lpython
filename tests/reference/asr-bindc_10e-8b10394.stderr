semantic error: The struct in c_p_pointer must be C interoperable
 --> tests/errors/bindc_10e_mod.py:5:1 - 7:10
  |
5 |    class S:
  |    ^^^^^^^^...
...
  |
7 |        b: i64
  | ...^^^^^^^^^^ not C interoperable
  |
5 |    class S:
  |    ~~~~~~~~...
...
  |
7 |        b: i64
  | ...~~~~~~~~~~ help: add the @ccallable decorator to this struct to make it C interoperable
